src/components/PageHeader/PageHeader.styled.tsx

```diff
@@ -1,8 +1,8 @@
 import theme from "@root/theme";
 import styled from "styled-components";
 
-export const StyledDrawerHeader = styled.div`
-	background-color: ${theme.newColors.grey2["100"]};
+export const StyledPageHeader = styled.div`
+	//background-color: ${theme.newColors.grey2["100"]};
```

Comment:
Not sure if this was commented by accident but I feel this line should be deleted changed to white